overrides/assets/stylesheets/home.css

```diff
@@ -516,6 +516,61 @@ footer nav.md-footer__inner {
   padding: 0;
 }
 
-footer .md-footer-meta .md-footer-meta__inner {
+.md-footer-meta{
   padding: 1em 0;
+}
+.md-footer-copyright{
+  margin-right: 18px;
+}
+@media (max-width: 1024px){
+.md-footer-copyright{
+  margin: auto;
+  text-align: center;
+}
+.md-social{
+  margin: auto;
+}
+}

```

Comment:
nit: can you add spacing between the classes here and fix the indentation so that this is easier to read?